project_template/models.py

```diff
@@ -292,6 +292,8 @@ class OwnedIncomeFromInvestmentsTable(models.Model):
 # Tabel Venituri investitii - actual row information
 class OwnedIncomeFromInvestmentsTableEntry(CommonIncomeFields):
     table = models.ForeignKey(OwnedIncomeFromInvestmentsTable, on_delete=models.CASCADE, null=True)
+    receiver_of_goods = models.ForeignKey(Person, on_delete=models.CASCADE, null=True)
+    position = models.CharField("Pozitia detinuta", max_length=128)
```

Comment:
See comment regarding the `position` variable in `forms.py`. This is redundant. Please, remove it.